superset-frontend/src/common/hooks/usePrevious/usePrevious.test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { renderHook } from '@testing-library/react-hooks';
+import { usePrevious } from './usePrevious';
+
+test('get undefined on frist render when initalValue is not defined', () => {
+  const hook = renderHook(() => usePrevious('state'));
+  expect(hook.result.current).toBeUndefined();
+});
+
+test('get initail value on frist render when initalValue is defined', () => {
```

Comment:
get initial value on the first render when initialValue is defined

Reply:
tnks 2 rsrs